test/core/relaxed-simd/i32x4_relaxed_trunc.wast

```diff
@@ -50,13 +50,15 @@
                        (v128.const f32x4 0 -1.0 4294967040.0 4294967296.0))
                ;; out of range -> saturate or UINT32_MAX
                (either (v128.const i32x4 0 0 4294967040 0xffffffff)
-                       (v128.const i32x4 0 0xffffffff 4294967040 0xffffffff)))
+                       (v128.const i32x4 0 0xffffffff 4294967040 0xffffffff)
+                       (v128.const i32x4 0 0xffffffff 4294967040 0)))
 
 (assert_return (invoke "i32x4.relaxed_trunc_f32x4_u"
                        (v128.const f32x4 nan -nan nan:0x444444 -nan:0x444444))
                ;; nans -> 0 or UINT32_MAX
                (either (v128.const i32x4 0 0 0 0)
-                       (v128.const i32x4 0xffffffff 0xffffffff 0xffffffff 0xffffffff)))
+                       (v128.const i32x4 0xffffffff 0xffffffff 0xffffffff 0xffffffff)
+                       (v128.const i32x4 0x80000000 0x80000000 0x80000000 0x80000000)))
```

Comment:
0x800000 comes from VCVTTPS2UDQ?
Think will be nice to comment here to say which cases lead to which results.
I should have documented it more properly in the first place.

Reply:
I thought about that but I cannot refer the algorithm/recipe by name, or issue that defines these. I'll just refer SM and V8 as source.

Reply:
actually it is different on V8, it looks like `c0000000, c0000000, c4444400, c4444400`. SM has `0x8000000`s

Reply:
What is SM's implementation of this?
V8 is https://source.chromium.org/chromium/chromium/src/+/refs/heads/main:v8/src/codegen/shared-ia32-x64/macro-assembler-shared-ia32-x64.h;l=769;drc=2450f2f5d0ce0da9b8cf493c533f9528ff17bab6